rclpy_examples/listener_qos_py.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2016 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import sys
+
+import rclpy
+from rclpy.qos import qos_profile_default, qos_profile_sensor_data
+
+from std_msgs.msg import String
+
+
+def chatter_callback(msg):
+    print('I heard: [%s]' % msg.data)
+
+
+def main(argv=sys.argv[1:]):
+    parser = argparse.ArgumentParser(formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument(
+        '-r', '--reliability', type=int, default=0, choices=[0, 1],
+        help='1: reliable, 0: best effort')
```

Comment:
Are there any other options planned? If not I would suggest using a boolean flag `--reliable` instead.